packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/validation.tsx

```diff
@@ -136,7 +136,9 @@ export const getMaxMin = (
           let max = BigNumber.minimum(
             method.limits.max,
             isSddFlow ? sddLimit.max : pair.buyMax,
-            isSddFlow ? sddLimit.max : limitMaxAmount
+            isSddFlow
+              ? sddLimit.max
+              : convertBaseToStandard('FIAT', limitMaxAmount, false)
```

Comment:
we need to convert it to proper amount to be minor string

Reply:
Could you make it as a variable?

Reply:
yeah I would love to refactor entire file since it's wild one :(